server.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const webpack = require('webpack');
+const webpackDevMiddleware = require('webpack-dev-middleware');
+
+const app = express();
+const config = require('./webpack.config.js');
+
+const compiler = webpack(config);
+
+// Tell express to use the webpack-dev-middleware and use the webpack.config.js
+// configuration file as a base.
+app.use(
+  webpackDevMiddleware(compiler, {
+    publicPath: config.output.publicPath,
+  }),
+);
+
+// Serve the files on port 3000.
+app.listen(3000, () => {
+//   console.log('Example app listening on port 3000!\n');
```

Comment:
### [Optional]
- I would recommend removing the commented code as removing would make your codebase clean, as it is only one comment I am approving your request just make sure to remove the commented code before merging. 🙂